src/merge.c

```diff
@@ -545,6 +545,7 @@ merge_iterator_create(platform_heap_id hid,
                      == ARRAY_SIZE(merge_itor->ordered_iterators),
                   "size mismatch");
 
+   platform_memfrag memfrag_merge_itor;
```

Comment:
Another idiomatic usage example:

To allocate memory for a field named `merge_itor`, declare an on-stack `platform_memfrag memfrag_merge_itor`, which is used by the allocator macro.